src/css/style.css

```diff
@@ -1,19 +1,44 @@
 /* FEEL FREE TO ADD STYLING */
+:root {
+  --bg: #dfdfdf;
+  --primary: #3182a6;
+  --danger: #ef5350;
+  --white: #fff;
+  --light-grey: #B0B0B0;
+  --hint-grey: #EFEFEF;
+  --primary-gradient: linear-gradient(166.21deg, #2a8c92 2.56%, #347bb2 122.3%);
+  --s-border-radius: 3px;
+  --l-border-radius: 8px;
+  --base-font: 16px;
+  --font: 'Roboto', sans-serif;
+  --drop-shadow: 0px 4px 12px rgba(42, 101, 146, 0.4);
+}
```

Comment:
Excellent addition showing how `root` can be used to declare global variables in CSS.

Reply:
I had to do with what I got and CSS variables were more viable than introducing a whole build system just for sass. Plus it allows easy theme edit like a dark theme and the likes.